letsencrypt_transip_nl/tests/test_letsencrypt_transip_nl.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+
+
+class TestLetsencryptTransipNl(TransactionCase):
+    def test_letsencrypt_transip_nl(self):
```

Comment:
If you plan the tests of the base module accordingly, you can here simply inherit those tests and rerun with a mock version of the transip library

Reply:
Ok, but what is there to test in this module though besides raising coverage?

Reply:
you mock the transip library and check it's called with the correct values